CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.5.0 - 2019-02-16
+
+### Misc
+
+- Reworked flashing logic with etcher-sdk
+- Upgrade to Electron v3.
+- Upgrade to NPM 6.7.0
```

Comment:
Not sure it's worth mentioning npm as it only affects us.

Reply:
Us, and anyone that decides to contribute to Etcher, so yes it's worth imho